Sources/SentryCrash/Recording/Tools/SentryCrashPlatformSpecificDefines.h

```diff
@@ -42,4 +42,8 @@ typedef struct nlist nlist_t;
 #    define LC_SEGMENT_ARCH_DEPENDENT LC_SEGMENT
 #endif /* __LP64__ */
 
+#ifndef SEG_DATA_CONST
```

Comment:
`l`: It would be helpful to have a comment here explaining the purpose of having `SEG_DATA_CONST` instead of using `"__DATA_CONST"` directly